[OSX] main.xib not compiled with Xcode 10.2

### Environment

* **FreeOrion Version**: master branch 0d6b28b142e0fa152ca52f089a0218bd34f91ece
* **Operating System**: MacOS Mojave 10.14.4
* **IDE/Toolchain**:
  * cmake 3.14.0
  * Xcode 10.2.1
* **Graphic card used**: not applicable
* **Fetched as** <!-- delete lines below that don't apply -->
  * Compiled from source


### Description

The build of FO fails with:

```
CompileXIB Xcode/main.xib
    cd /Users/adrian/Projekte/freeorion/freeorion.git
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --module FreeOrion --output-partial-info-plist /Users/adrian/Projekte/freeorion/freeorion.clang/FreeOrion.build/Release/freeorion.build/main-PartialInfo.plist --auto-activate-custom-fonts --target-device mac --minimum-deployment-target 10.9 --output-format human-readable-text --compile /Users/adrian/Projekte/freeorion/freeorion.clang/Release/FreeOrion.app/Contents/Resources/main.nib /Users/adrian/Projekte/freeorion/freeorion.git/Xcode/main.xib
/* com.apple.ibtool.document.warnings */
/Users/adrian/Projekte/freeorion/freeorion.git/Xcode/main.xib:global: warning: This file is set to build for a version older than the deployment target. Functionality may be limited.
/* com.apple.ibtool.document.errors */
/Users/adrian/Projekte/freeorion/freeorion.git/Xcode/main.xib:global: error: Compiling for earlier than macOS 10.6 is no longer supported.

** BUILD FAILED **
```

### Expected Result
Successful compile


### Steps to reproduce

* `cmake -GXcode -DCMAKE_INSTALL_PREFIX=INSTALL ../freeorion.git/`
* `cmake --build . --config Release -- -target install`

@Vezzra any idea what is going on there? Also what do you use as build environment?

--- a/Xcode/main.xib
+++ b/Xcode/main.xib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="5056" systemVersion="13E28" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none">
     <dependencies>
-        <deployment version="1050" defaultVersion="1070" identifier="macosx"/>
+        <deployment version="1070" identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="5056"/>
     </dependencies>
     <objects>
@@ -11,7 +11,7 @@
             </connections>
         </customObject>
         <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
-        <customObject id="-3" userLabel="Application"/>
+        <customObject id="-3" userLabel="Application" customClass="NSObject"/>
         <customObject id="373" customClass="NSFontManager"/>
         <menu title="AMainMenu" systemMenu="main" id="29" userLabel="MainMenu">
             <items>

